src/db/homeseeker/schedule.ts

```diff
@@ -0,0 +1,113 @@
+import type { ResultSetHeader, RowDataPacket } from "mysql2";
+
+import { pool } from "@/db/index";
+
+export interface Schedule extends RowDataPacket {
+	id: number;
+	property_id: number;
+	start: Date;
+	end: Date;
+}
+
+/**
+ * Retrieves all schedules made from database.
+ */
+export async function getSchedules(): Promise<Schedule[]> {
+	const [res] = await pool.execute<Schedule[]>(
+		"SELECT * FROM bookings_db.hs_schedule",
```

Comment:
It's generally considered bad practice to use `SELECT *` for data-fetching queries since you are (a) hiding the columns away from anybody reading the file, and (b) relying on the schema not changing. Instead, it's best to list out all of the columns you need, even if it is all of them.